content/common/navigation/engine/guides.yaml

```diff
@@ -245,6 +245,8 @@ navigation:
                 path: /luau/functions
               - title: Type Coercion
                 path: /luau/type-coercion
+          - title: Luau Type Checking
```

Comment:
Would type checking fit better under the features section?

Reply:
This one thing I'm going to ignore because I feel like its better for Roblox to deicde on this one

I've been busy over the weekend so haven't had a time to address the new reviews, again, thanks for cleaning up my 10/10 english